docs/release-notes/6.7.md

```diff
@@ -0,0 +1,32 @@
+# Release notes for Gluster 6.7
+
+This is a bugfix release. The release notes for [6.0](6.0.md), [6.1](6.1.md),
+ [6.2](6.2.md), [6.3](6.3.md), [6.4](6.4.md), [6.5](6.5.md) and [6.6](6.6.md)
+contains a listing of all the new features that were added
```

Comment:
'contain' a listing of all the new features that have been added along with bugs fixed in the GlusterFS 6 stable release.

Reply:
done